api/HTMLImageElement.json

```diff
@@ -1063,10 +1063,10 @@
               "version_added": "41"
             },
             "safari": {
-              "version_added": "11.1"
+              "version_added": "14"
```

Comment:
Checks out, this was also recent: https://trac.webkit.org/changeset/257572/webkit